libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -567,6 +851,41 @@ void AP_UAVCAN::do_cyclic(void)
 
                 rc_out_sem_give();
             }
+
+            uint64_t c_send_time = AP_HAL::millis64();
+
+            if (fix_out_array[_uavcan_i] != nullptr) {
+                if ((c_send_time - fix_out_last_send_time) > abs(1000 / _broadcast_fix_rate)) {
```

Comment:
`_broadcast_fix_rate` could be 0, and would result in div by 0.

Reply:
Done